### PR TITLE
12.0 labelized structure form new fields

### PR DESCRIPTION
**Pull request by houssine78**
Adding fields to the structure

### CODE REVIEW

File: investor_wallet_platform_base/data/sector_area_data.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<odoo>
+	<data noupdate="1">
+		<record id="res_partner_industry_iwp_A" model="res.partner.industry">
+	        <field name="name">Agriculture</field>
```
Comment (robinkeunen) [inline investor_wallet_platform_base/data/sector_area_data.xml:4]: Shouldn't it be in English?

File: investor_wallet_platform_base/models/partner.py [hunk 2]
```diff
@@ -31,63 +33,101 @@ class ResPartner(models.Model):
...
+                }
+            operation_sequence = ir_sequence_obj.create(sequence_vals)
+            sequence_vals = {
+                'name': 'Account Default Subscription Journal ' + self.name,
+                'padding': 3,
+                'use_date_range': True,
+                'prefix': 'SUBJ/%(year)s/',
+                }
+            journal_sequence = ir_sequence_obj.create(sequence_vals)
+            # TODO create journal
```
Comment (robinkeunen) [inline investor_wallet_platform_base/models/partner.py:116]: looks like it's done.

File: investor_wallet_platform_base/views/activity_area_view.xml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	    </field>
+	</record>
+	
+	<record id="view_activity_area_tree" model="ir.ui.view">
+	    <field name="name">activity.area.tree</field>
+	    <field name="model">activity.area</field>
+	    <field name="arch" type="xml">
+	    	<tree string="Activity areas">
+	        	<field name="name"/>
+	   			<field name="country_id"/>
```
Comment (robinkeunen) [inline investor_wallet_platform_base/views/activity_area_view.xml:25]: Configure your IDE to use spaces.

**Review by robinkeunen — APPROVED** [review]
Except a few minor comments, LGTM.

File: investor_wallet_platform_base/data/sector_area_data.xml (hunk 1 above)
Comment (houssine78) [reply investor_wallet_platform_base/data/sector_area_data.xml:4]: yes it should ! done

File: investor_wallet_platform_base/views/activity_area_view.xml (hunk 3 above)
Comment (houssine78) [reply investor_wallet_platform_base/views/activity_area_view.xml:25]: I will i promess !

**Pull request merged**